keyboards/ah/haven80_hotswap/config.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 Freather
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define EECONFIG_USER_DATA_SIZE 4
```

Comment:
Is there a reason for this?

Reply:
> Is there a reason for this?

this is for using the eeconfig_read_user_datablock in the via/keymap.c. if this is not included in the config, the corresponding code will not work